system/jlib/jsecrets.cpp

```diff
@@ -1349,18 +1402,10 @@ IPropertyTree *createIssuerTlsClientConfig(const char *issuer, bool acceptSelfSi
     return info.getClear();
 }
 
-IPropertyTree *getIssuerTlsServerConfig(const char *name)
+static ISyncedPropertyTree * createIssuerTlsServerConfig(const char *name)
```

Comment:
I know it's existing code (but since you are changing these).. I've been slowly changing the "name" parameter on these routines to "issuer" to be more self documenting.  Or if you prefer "issuerName".

The issuer which is part of the path to the secret act like categories for the various trust zones: public, local, remote, etc.